src/Layout/tool_fsharp/tool_fsc.csproj

```diff
@@ -5,13 +5,28 @@
     <ServerGarbageCollection>true</ServerGarbageCollection>
   </PropertyGroup>
 
+  <!-- MIBC, PGO -->
+  <Import Project="..\redist\targets\GetRuntimeInformation.targets" /> <!-- Rid is set here -->
+  <PropertyGroup>
+    <MibcPath>$(PkgMicrosoft_FSharp_Compiler)/contentFiles/mibc</MibcPath>
+    <MibcRid>$(Rid)</MibcRid>  
+    <MibcRid Condition="$(MibcRid.Contains('musl-'))">$(MibcRid.Replace('musl-', ''))</MibcRid>
```

Comment:
Is this valid @mmitche? How portable are PGO profiles, can they float across different libc ABIs?

Reply:
Not sure. I'm going to draw a random name out of the hat on that one and tag @agocke for general knowledge.

Reply:
Maybe @EgorBo?

Reply:
Sorry for the delayed response, Mibc are pretty portable, in the worst case a non-native profile might lead to suboptimal codegen. I think even for dotnet/runtime BCL for osx we still use MIBC collected for Linux.

Reply:
Thanks for the explanation!